ethstore/src/json/bytes.rs

```diff
@@ -0,0 +1,58 @@
+use std::{ops, str};
+use serde::{Deserialize, Deserializer, Error, Serialize, Serializer};
+use rustc_serialize::hex::{ToHex, FromHex, FromHexError};
+
+#[derive(Debug, PartialEq)]
+pub struct Bytes(Vec<u8>);
```

Comment:
They share the same name, but they are not the same. `json::Bytes` deserialization is extremely lenient, `rpc::Bytes` deserialization allows `0x` prefixes. Here, the format is very strict